### ISSUE TITLE
artifacts not getting logged

### ISSUE THREAD

**Issue by karenc3** [pycaret/pycaret#421]
Hi,

I'm just trying out the code in the `PyCaret 2 Classification.ipynb` notebook. I've setup the mlflow tracking server using `mlflow.set_tracking_uri()`

While checking the mlflow runs: metrics, params and tags are getting logged on the server. Everything except artifact. The artifact folder is empty.

This is the `setup()` function code I'm using

```
data = get_data('juice', profile=True)
setup(data, target = 'Purchase', session_id=123, log_experiment=True, experiment_name=experiment_name, log_plots=True, log_profile=True, log_data=True, profile=True, verbose=True)
```

For installing `pycaret` I used: `pip install pycaret==2.0`

**Comment by pycaret** [comment]
@karenc3 Yes. you are right. `log_experiment` controls all logging. Can you check if without setting back-end you can see artifacts on `localhost:5000`

**Comment by karenc3** [comment]
Thank you for your reply! 

You mean the `--backend-store-uri` parameter while starting the mlflow server right? I tried disabling it but still no artifacts are getting logged in the artifact folder.

**Comment by moezali1** [comment]
Closing as part of the 4.0 backlog cleanup — see #4177.

PyCaret 4.0 is a ground-up rewrite (OOP-only API, sklearn-native engine, Python 3.11+, sklearn ≥ 1.7) and almost all pre-4.0 issues either reference removed APIs (`setup()`, `plot_model`, `evaluate_model`, `interpret_model`, `automl`, the legacy time-series module, the Streamlit dashboard, …) or no longer reproduce on the new code path.

**If this still affects you in 4.0, please re-file as a new issue** with:

- the experiment constructor call (`Experiment(...)`),
- the operation that fails,
- the full traceback,
- your Python / sklearn / pycaret versions.

Resources for 4.0:

- Docs: https://pycaret.org/docs
- Migrate from 3.x: https://pycaret.org/docs/guides/migrating-from-3-x
- API reference: https://pycaret.org/reference
- FAQ: https://pycaret.org/docs/resources/faqs

Thanks for filing — and apologies for the long latency. The repo had drifted; we're starting clean.
